lib/modules/versioning/hermit/index.spec.ts

```diff
@@ -0,0 +1,210 @@
+      ${'17.2.3a1'}   | ${17}   | ${2}    | ${3}
+      ${'17.2.3-foo'} | ${17}   | ${2}    | ${3}
+      ${'17.2.3+m1'}  | ${17}   | ${2}    | ${3}
+      ${'@17'}        | ${17}   | ${null} | ${null}
+      ${'@17.2'}      | ${17}   | ${2}    | ${null}
+      ${'@stable'}    | ${null} | ${null} | ${null}
+    `(
+      'getMajor, getMinor, getPatch for "$version"',
+      ({ version, major, minor, patch }) => {
+        expect(versioning.getMajor(version)).toBe(major);
+        expect(versioning.getMinor(version)).toBe(minor);
+        expect(versioning.getPatch(version)).toBe(patch);
+      }
+    );
+  });
+
+  describe('equals', () => {
+    test.each`
+      version       | other        | expected
+      ${'1'}        | ${'1.2'}     | ${false}
+      ${'@1'}       | ${'@1.2'}    | ${false}
+      ${'@1.2'}     | ${'@1.2'}    | ${true}
+      ${'@1.2'}     | ${'@1.3'}    | ${false}
+      ${'@1.2.3'}   | ${'@1.2'}    | ${false}
+      ${'@1.2.3_4'} | ${'@1.2.3'}  | ${false}
+      ${'@latest'}  | ${'@1'}      | ${false}
+      ${'@stable'}  | ${'@stable'} | ${true}
+      ${'stable'}   | ${'stable'}  | ${true}
+    `(
+      'equals("$version", "$other") === $expected',
+      ({ version, other, expected }) => {
+        expect(versioning.equals(version, other)).toBe(expected);
+      }
+    );
+  });
+
+  describe('matches', () => {
+    test.each`
+      version      | other        | expected
+      ${'@1'}      | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.2'}    | ${true}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${false}
+      ${'@latest'} | ${'@1'}      | ${false}
+      ${'@stable'} | ${'@stable'} | ${true}
+    `(
+      'matches("$version", "$other") === $expected',
+      ({ version, other, expected }) => {
+        expect(versioning.matches(version, other)).toBe(expected);
+      }
+    );
+  });
+
+  describe('isGreaterThan', () => {
+    test.each`
+      version      | other        | expected
+      ${'@1'}      | ${'@1.2'}    | ${true}
+      ${'@1.2'}    | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.3'}    | ${false}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${false}
+      ${'1.2.3'}   | ${'@latest'} | ${true}
+      ${'@latest'} | ${'@1'}      | ${false}
+      ${'@stable'} | ${'@latest'} | ${true}
+      ${'@latest'} | ${'@stable'} | ${false}
+    `(
+      'isGreaterThan("$version", "$other") === $expected',
+      ({ version, other, expected }) => {
+        expect(versioning.isGreaterThan(version, other)).toBe(expected);
+      }
+    );
+  });
+
+  describe('isLesserThanRange', () => {
+    test.each`
+      version      | other        | expected
+      ${'@1'}      | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.2'}    | ${false}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${true}
+      ${'@latest'} | ${'@1'}      | ${true}
+      ${'@stable'} | ${'@latest'} | ${false}
+      ${'@latest'} | ${'@stable'} | ${true}
+    `(
+      'isLessThanRange("$version", "$other") === $expected',
```

Comment:
```suggestion
      '("$version", "$other") === $expected',
```
already in parent description

Reply:
instead of removing the name of the functions here, how about I just follow style with other tests to get rid of the parent describe(`<function_name>`, () => {})

@viceice